src/adapters/collateralized_simple_interest_loan_adapter.ts

```diff
@@ -339,4 +451,129 @@ export class CollateralizedSimpleInterestLoanAdapter implements Adapter.Interfac
             );
         }
     }
+
+    /**
+     * Collateral is seizable if the collateral has not been withdrawn yet, and the
+     * loan has been defaulted for the duration of the grace period.
```

Comment:
Please change "has been defaulted..." to "has been in a state of default for a duration of time greater than the grace period."